src/components/tooltips/UpgradeTooltip.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import Link from 'next/link';
+import { BiCrown } from 'react-icons/bi';
+
+import Tooltip from './Tooltip';
+
+export default function UpgradeTooltip(): JSX.Element {
+  return (
+    <Tooltip
+      id="upgrade-tooltip"
+      className="max-w-80"
+      clickable
+    >
+      <h2 className="flex font-normal items-center justify-center text-cyan-600">
+        <BiCrown className="mr-1 font-thin" />
+        Premium
+      </h2>
+      <div className="text-center">
+        Auto saving to persistent storage only available for premium users.
+      </div>
+      <Link
+        href="https://maffin.io"
```

Comment:
will need to update once there is a "manage your subscription" page or a pricing table